src/Stepper/Stepper.js

```diff
@@ -23,9 +23,9 @@ class Stepper extends Component {
      */
     activeStep: PropTypes.number,
     /**
-     * Should be two or more `<Step />` components
+     * Should be two or more `<Step />` components, but also accepts one.
```

Comment:
In that case I think the best solution would be to leave the description as it is on master and revert this part of the change `Should be two or more `<Step />` components`. Keyword being _should_, instead of _must_.